TestNG - changes.

Radu-Stroe opened the pull request:
Change several outdated WebElements - Homepage.
Change several outdated WebElements, create a loginMenu WebElement - LoginPage.
Change myNewsLettersButton WebElement - MyAccountPage.
Change several outdated WebElements, change order of objects - NavigationPage.
Change myDetailsButton WebElement - PersonalDetailsPage.
Change several outdated WebElements - RegisterCompletePage.
Set priority for tests.

Radu-Stroe committed:
Change order of objects - DataProviders.
Change several outdated WebElements - Homepage.
Change several outdated WebElements, create a loginMenu WebElement - LoginPage.
Change myNewsLettersButton WebElement - MyAccountPage.
Change several outdated WebElements, change order of objects - NavigationPage.
Change myDetailsButton WebElement - PersonalDetailsPage.
Change several outdated WebElements - RegisterCompletePage.
Set priority for tests.

Radu-Stroe committed:
PR Quality Review changes.

cristinanedin commented on src/test/java/internship/tests/PersonalDetailsTest.java:
```diff
@@ -6,19 +6,18 @@
+    private final String TEST_EMAIL = "radu.telegraph+10@gmail.com";
     private final String TEST_PASSWORD = "qwer1234";
-    private final String TEST_NEW_EMAIL = "radu.telegraph+12@gmail.com";
+    private final String TEST_NEW_EMAIL = "radu.telegraph+11@gmail.com";
```
this doesn't look maintainable

Radu-Stroe replied:
Noted and fixed.

cristinanedin commented on src/test/java/internship/tests/NavigationPageTest.java:
```diff
@@ -11,11 +11,10 @@
             description = "Open Navigation page and verify sections",
-            dataProvider = "sectionNavigation", dataProviderClass = DataProviders.class)
+            dataProvider = "sectionNavigation", dataProviderClass = DataProviders.class,
+            priority = 2)
```
why you need this?

Radu-Stroe replied:
It seems that I don't need priority for my tests. Fixed.

Radu-Stroe committed:
Remove termsAndConditions WebElement and method - CustomerSubscribePage.
Add termsAndConditions WebElement and method - Homepage.
Create 2 more WebElement for unsubscribeNewsletters, change the uncheckNewsletters method - MyNewslettersPage.
Change buttonNews WebElement - NavigationPage.
Change myDetailsButton WebElement - PersonalDetailsPage.
Remove priority from all tests.
Change access modifier to private for all constants - CustomerSubscribePageTest.
Add uncheckNewsletters method - MyNewslettersTest.
Modify tests to be reusable - PersonalDetailsTest.
Navigate to TermsAndConditionsPage from Homepage - TermsAndConditionsTest.
